### PR TITLE
small changes to two of the Info articles

### PR DESCRIPTION
**Pull request by Zacatero**

### CODE REVIEW

**Review by byemc — COMMENTED** [review]
@newbold maybe it might be worth creating a new page for the Discord

File: info/Community/irc.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ You could use something else, if you’d like (anything will work as long as it
 
 ### Channels
 
-Here are some of the popular channels on the server:
+Here are some of the popular channels on the server (these are also the channels bridged to Discord):
```
Comment (byemc) [inline info/Community/irc.md:46]: Maybe move this to the **Discord** heading?

**Review by byemc — CHANGES_REQUESTED** [review]

File: info/Community/irc.md (hunk 1 above)
Comment (Zacatero) [reply info/Community/irc.md:46]: That would be up to @newbold if he wants to have a bigger Discord section. I was just going with what was already there.

**Comment by newbold** [comment]
Thanks for the suggestions here! I think these two articles are OK as they are. 👍

**Pull request closed without merging**